internal/services/jsonrpc/server.go

```diff
@@ -290,7 +290,7 @@ func NewServer(addr string, log log.Logger, opts ...Opt) (*Server, error) {
 	h = realIPHandler(h, cfg.proxyCount) // for effective rate limiting
 	h = recoverer(h, log)                // first, wrap with defer and call next ^
 
-	mux.Handle("POST "+pathRPCV1, h)
```

Comment:
@Yaiba Turns out the new router magic was blocking `OPTIONS`. :(

Reply:
ah, you're right, this `POST /xx` will only accept POST..